### ISSUE TITLE
[Feature request] Can EBCDIC code page support be added to NP++?

### ISSUE THREAD

**Issue by pjfarleyiii** [notepad-plus-plus/notepad-plus-plus#15156]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Description of the Issue

I work on an IBM mainframe and it is sometimes helpful to be able to edit a mainframe EBCDIC file on my PC without EBCDIC -> ISO8859-1 translation.  Is it possible to support viewing and editing EBCDIC-encoded files in at least a few of the more common code pages like IBM1047, IBM037, IBM1140, etc.?  Not the double-byte code pages, just the single-byte ones.

### Describe the solution you'd like.

Options under the Encoding menu item to support EBCDIC encoded text files.

### Debug Information

```shell
Notepad++ v8.6.7   (64-bit)
Build time : May 12 2024 - 20:04:00
Path : C:\Program Files\Notepad++\notepad++.exe
Command Line : 
Admin mode : OFF
Local Conf mode : OFF
Cloud Config : OFF
Periodic Backup : ON
OS Name : Windows 10 Pro (64-bit)
OS Version : 22H2
OS Build : 19045.4412
Current ANSI codepage : 1252
Plugins : 
    mimeTools (3.1)
    NppConverter (4.6)
    NppExport (0.4)
```

### Anything else?

https://en.wikipedia.org/wiki/EBCDIC

**Comment by alankilborn** [comment]
Non-unicode encodings in Notepad++ are really only supported because they were 20+ years ago in Notepad++ 's infancy.  They don't get much if any attention these days.  So I'd doubt that EBCDIC will get any attention.  But, good luck.

**Comment by rkroemer** [comment]
I found two very old topics in Notepad++ Community regarding this topic as well - unfortunately, they're both tending in the same direction that there's no solution (yet)

- [Hex/EBCDIC edit mode](https://community.notepad-plus-plus.org/topic/14837/hex-ebcdic-edit-mode) (referring to the one below)
- [Looking for an ebcdic plugin?](https://community.notepad-plus-plus.org/topic/14136/looking-for-an-ebcdic-plugin)
